js/admin.js

```diff
@@ -0,0 +1,1019 @@
+
+    let selectCategories = document.createElement('select');
+    selectCategories.setAttribute('id', deleteOrEdit == 'delete' ? 'delete-product-select' : 'edit-oriduct-select');
+
+    let selectCategoriesOptions = document.createElement('option');
+    if (oldProducts.length !== 0) {
+        selectCategoriesOptions.innerHTML = 'Choose a product';
+    }
+    else {
+        selectCategoriesOptions.innerHTML = 'There is no product yet';
+    }
+
+    selectCategories.appendChild(selectCategoriesOptions);
+
+    oldProducts.forEach(element => {
+        selectCategoriesOptions = document.createElement('option');
+        selectCategoriesOptions.innerHTML = element.name;
+        selectCategoriesOptions.setAttribute('data-id', element.id);
+
+        selectCategories.appendChild(selectCategoriesOptions);
+    });
+
+    formGroup.appendChild(selectCategories);
+    form.appendChild(formGroup);
+
+    let deleteBtn = document.createElement('button');
+    deleteBtn.setAttribute('type', 'submit');
+
+    if (deleteOrEdit == 'edit')
+        deleteBtn.style.color = "var(--primary-text-color)";
+    deleteBtn.classList.add('btn', deleteOrEdit === 'delete' ? 'btn-danger' : 'btn-warning');
+    deleteBtn.innerHTML = deleteOrEdit === 'delete' ? 'Delete Category' : 'Edit Category';
+
+    form.appendChild(deleteBtn);
+
+    categoriesContainer.appendChild(form);
+}
+
+// Add product
+function addProducts(productName) {
+
+    // Add Notification
+
+}
+
+// Delete Product
+if (window.location.href.indexOf('/admin-delete-product.html') !== -1) {
+
+    // Remove Product Frm DB
+    function removeProductFromDB(id) {
+        let oldProducts = localStorage.getItem('products');
+        oldProducts = JSON.parse(oldProducts);
+
+        oldProducts.forEach((element, index) => {
+            if (element.id == id) {
+                oldProducts.splice(index, 1);
+
+                // Add Notification 
+                addNotificationToDB(`Delete Product "${element.name}" from system`);
+            }
+        });
+
+        oldProducts = JSON.stringify(oldProducts);
+        localStorage.setItem('products', oldProducts);
+
+    }
+
+    // Remove product
+    (function removeProducts() {
+        showProducts('delete');
+
+        let deleteBtn = document.querySelector('button.btn-danger');
+        let selectedProduct = document.querySelector('#delete-product-select');
+        deleteBtn.addEventListener('click', (e) => {
+            e.preventDefault();
+            if (document.querySelector('#delete-product-select').value == 'Choose a product') {
+                alert('select the product');
+                return;
+            }
+            let productId = selectedProduct.options[selectedProduct.selectedIndex];
+
+            removeProductFromDB(productId.getAttribute('data-id'));
+
+            productId.remove();
+        });
+    }());
+
+}
+
+function editProducts(productId) {
+
+    // Add Notification
+
+}
```

Comment:
Why this function is empty remove it if you'll not use it

Reply:
If we have time we gonna handle it. As we have decided

Reply:
ok